test/vstest.console.UnitTests/ExecutorUnitTests.cs

```diff
@@ -131,7 +131,7 @@ public void ExecutorWithInvalidArgsShouldPrintErrorMessage()
 
             Assert.AreEqual(1, exitCode, "Exit code must be One when no arguments are provided.");
 
-            Assert.IsTrue(mockOutput.Messages.Any(message => message.Message.Contains(string.Format(CommandLineResources.TestSourceFileNotFound, badArg))));
+            Assert.IsTrue(mockOutput.Messages.Any(message => message.Message.Contains(string.Format(CommandLineResources.InvalidArgument, badArg))));
```

Comment:
Need more tests to cover -- or - as well
Also, add the argument such as /badArgument:SomeValue

Reply:
Covered Tests for 
i) for the argument - and
ii)for the argument /badArgument:abc
Kindly have a look and share your feedback